packages/suite/src/components/suite/FormattedNftAmount.tsx

```diff
@@ -1,89 +1,127 @@
-import styled from 'styled-components';
+import { useTheme } from 'styled-components';
 
 import { SignValue } from '@suite-common/suite-types';
-import { getNftTokenId } from '@suite-common/wallet-utils';
+import { getNftTokenId, isNftMultitokenTransfer } from '@suite-common/wallet-utils';
 import { TokenTransfer } from '@trezor/connect';
-import { variables } from '@trezor/components';
+import { Column, Flex, Row, Text } from '@trezor/components';
 
-import { HiddenPlaceholder, Sign } from 'src/components/suite';
+import { HiddenPlaceholder, Sign, Translation } from 'src/components/suite';
 // importing directly, otherwise unit tests fail, seems to be a styled-components issue
 import { TrezorLink } from 'src/components/suite/TrezorLink';
 import { useSelector } from 'src/hooks/suite/useSelector';
+import { useTranslation } from 'src/hooks/suite';
+import { BlurUrls } from 'src/views/wallet/tokens/common/BlurUrls';
 
-const Container = styled.div`
-    max-width: 100%;
-    display: flex;
-`;
-
-const Symbol = styled.div`
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    max-width: 120px;
-`;
-
-const StyledTrezorLink = styled(TrezorLink)`
-    color: ${({ theme }) => theme.legacy.TYPE_GREEN};
-    text-decoration: underline;
-    display: flex;
-    font-size: ${variables.FONT_SIZE.TINY};
-    line-height: initial;
-    align-items: center;
-`;
-
-const NoLink = styled.div`
-    display: flex;
-`;
-
-const Id = styled.div`
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    max-width: 145px;
-`;
+export const Id = ({ id }: { id?: string }) => {
+    return id ? (
+        <Text maxWidth="145px" ellipsisLineCount={1} textWrap="nowrap">
```

Comment:
nitpick: it's just my personal preference but if we use px we can use number.

Like:
```
maxWidth={145}
```

Reply:
Done